src/main/java/Example.java

```diff
@@ -0,0 +1,19 @@
+import org.springframework.boot.*;
+import org.springframework.boot.autoconfigure.*;
+import org.springframework.stereotype.*;
+import org.springframework.web.bind.annotation.*;
```

Comment:
One of the best practices is to reduce the number of imports with wildcard: http://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Sometimes they're useful, but not in this case. Let's follow the best practice here and import classes explicitly.

By the way, try to set cursor on line with import and press `Alt+Enter`, here should be an item for automatically replace import all by import individual classes.